src/jp2image.cpp

```diff
@@ -345,7 +346,7 @@ namespace Exiv2
                                     (rawData.pData_[0] == 'I' || rawData.pData_[0] == 'M')) {
                                     foundPos = true;
                                 } else {
-                                    const std::string exifHeader{"Exif\0\0", 6};
```

Comment:
I think it would be better this version:

```cpp
const std::array<char, 6> exifHeader{'E','x','i','f',0,0};
```
So that we do not need to have comments to understand the values.